recipe/meta.yaml

```diff
@@ -9,21 +9,16 @@ source:
   sha256: c0946ed31d71f1b732b5aaa6da5a0388a345019af232ce2f49c766e2d6795c51
 
 build:
-  number: 1
+  number: 2
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
     - pip
-
   run:
-    - python >=3.6
-    - numpy >=1.19
-    - scipy >=1.5,!=1.6.1
-    - matplotlib-base >=3.3
-    - pandas >=1.1
```

Comment:
Folks expecting these extras may be surprised with this change. Maybe we should to do a multiple outputs with a "core" package and another one with the extras?